packages/cli/src/tools/config/index.js

```diff
@@ -172,6 +182,23 @@ function loadConfig(projectRoot: string = process.cwd()): ConfigT {
     initialConfig,
   );
 
+  if (depsWithWarnings.length) {
+    logger.warn(
+      `The following packages use deprecated "rnpm" config that will stop working from next release:\n${depsWithWarnings
+        .map(
+          ([name, link]) =>
+            `  - ${chalk.bold(name)} ${chalk.dim(
+              `(${chalk.underline(link)})`,
```

Comment:
I recommend using: `name: url` instead of the parenthesis.

Reply:
so:
```
  - [bold appcenter]: [dimmed,underlined link]
```
?

Reply:
I like it.